stock_release_channel_partner_address/readme/CREDITS.md

```diff
@@ -0,0 +1,4 @@
+**Financial support**
+
+- Cosanum
+- Camptocamp R&D
```

Comment:
Do we need this ?

Reply:
I saw it in another module of the repo, I kept it :man_shrugging: